src/_setup_packages.sh

```diff
@@ -116,7 +116,7 @@ function install_docker() {
 function install_docker_compose() {
   info "upgrading docker-compose..."
   curl -sL $(curl -sL \
-    https://api.github.com/repos/docker/compose/releases/tags/v2.2.3 | jq -r \
+    https://api.github.com/repos/docker/compose/releases/latest | jq -r \
```

Comment:
I am working on a change to permit pinning via envvar and updating ad-hoc, but it is a larger lift so for now we can ship this and come back around in a later changeset.